bucket/butterflow.json

```
@@ -20,6 +20,10 @@
         "architecture": {
             "64bit": {
                 "url": "https://github.com/dthpham/butterflow/releases/download/$matchRel/butterflow-$version-win64.zip",
+                "hash": {
+                    "url": "https://raw.githubusercontent.com/dthpham/butterflow/master/README.md",
+                    "find": "$basename[\\S\\s]+\\* Sha256: $sha256"
```

Multilne regex is needed
```suggestion
                    "regex": "(?sm)$basename.*?\\*\\s+Sha256:\\s+$sha256"
```

---

`$basename\\s+\\* Sha256: $sha256` is enough.

---

Thanks, I don't have much experience with regular expressions.